Temporarily disable unused dependencies in StockFlow.jl and Rewrite.jl

NSepahrom1984 opened the pull request:
This pull request includes modifications `StockFlow.jl` and `Rewrite.jl` to temporarily disable dependencies and functionality that are not currently in use.

- The dependency on `AlgebraicRewriting` has been commented out in `Project.toml`
- The import of `FinSet` has been commented out in `StockFlow.jl`
- All function rewrite definitions inside `Rewrite.jl` have been commented out

These changes are intended to simplify the working version of the repository for integrating new functionality without breaking the current build. The commented code can be restored as needed later.

NSepahrom1984 committed:
Add final versions of modified files

Saityi reviewed (dismissed):
@NSepahrom1984 These modifications should probably replace the originals rather than being added in addition to them. 

I.e., `StockFlow.jl` should be in `src/`, and `Rewrite.jl` should be in `src/syntax/` rather than the root project directory.

We should also remove `AlgebraicRewriting` from `Project.toml`. I think you may have forgotten to commit your changes to it?

As-is, `StockFlow.jl` (the library) still depends on `AlgebraicRewriting.jl`.

NSepahrom1984 committed:
Remove old StockFlow.jl and Rewrite.jl from root directory

NSepahrom1984 commented:
Hi @Saityi,

I’ve made the requested changes:
- Moved `StockFlow.jl` to `src/`
- Moved `Rewrite.jl` to `src/syntax/`
- Removed `AlgebraicRewriting` from `Project.toml`

Please let me know if anything else is needed. Thank you!

NSepahrom1984 committed:
Fix filename for StockFlow.jl

NSepahrom1984 commented:
Hi @Saityi,  
I’ve fixed the filename and committed the changes.  
Now the workflows are pending approval.  
Please let me know if anything else is needed. Thank you!

Saityi commented:
Integrated into #145
